pkgs/firehose/lib/src/health/health.dart

```diff
@@ -92,6 +101,74 @@ Documentation at https://github.com/dart-lang/ecosystem/wiki/Publishing-automati
     );
   }
 
+  Future<HealthCheckResult> breakingCheck(Github github) async {
+    final repo = Repository();
+    final packages = repo.locatePackages();
+    var changeForPackage = <Package, BreakingChange>{};
+    var baseDirectory = Directory('../base_repo');
+    for (var package in packages) {
+      var currentPath =
+          path.relative(package.directory.path, from: Directory.current.path);
+      var basePackage = path.relative(
+        path.join(baseDirectory.absolute.path, currentPath),
+        from: currentPath,
+      );
+      print('Look for changes in $currentPath with base $basePackage');
+      var runApiTool = Process.runSync(
+        'dart-apitool',
+        [
+          'diff',
+          ...['--old', basePackage],
+          ...['--new', '.'],
+          ...['--report-format', 'json'],
```

Comment:
If this run finds breaking changes, should we re-run this report in another format and dump the results to stdout?